x/mongo/driver/cryptx/crypt_clientoptions.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) MongoDB, Inc. 2021-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package cryptx
+
+import (
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/x/mongo/driver"
+)
+
+// SetCrypt specifies a custom driver.Crypt to be used to encrypt and decrypt documents. The default is no encryption.
+// Using options.SetAutoEncryptionOptions will override anything set here.
```

Comment:
I think this sentence can be removed. `options.SetAutoEncryptionOptions` does not override the Crypt option.

Reply:
Removed.